completion_aggregator/api/v1/views.py

```diff
@@ -416,6 +426,28 @@ def get(self, request, course_key):
         )
         return paginator.get_paginated_response(serializer.data)
 
+    def get(self, request, course_key):
+        """
+        Handler for GET requests.
+        """
+        params = {}
```

Comment:
Could we just initialize it like `params = {'root_block': request.query_params.get('root_block', None)}` here?